flixel/FlxObject.hx

```diff
@@ -540,6 +549,17 @@ class FlxObject extends FlxBasic
 	 * Overriding this will force a specific color to be used for debug rect.
 	 */
 	public var debugBoundingBoxColor:Null<Int> = null;
+	
+	public var debugColorScheme(default, null):DebugColorScheme;
+	
+	/**
+	 * Can be overriden for RenderBlit mode to re-render stuff with new colors
+	 */
+	public function setDebugColorScheme(debugColorScheme:DebugColorScheme)
```

Comment:
of course it doesn't trigger it, but this is expected. Various classes have such behavior, for example you need to reset filters for display objects every time you change properties of one of the filters.
So this isn't bug in my opinion. We just need to document this behavior.

You can have the same "issue" with this code:
```
var colorScheme = { solid: FlxColor.RED, partial: FlxColor.GREEN, notSolid: FlxColor.BLUE };
object.setDebugColorScheme(colorScheme);
colorScheme.solid = Std.random(0xFFFFFF);
```
this example won't trigger update as well. So that's why i'm for original way (with `debugColorScheme` as a property with setter)